File: src/main/java/com/basecolon/uisless/model/DatabaseModel.java
package com.basecolon.uisless.model;

import com.google.gson.annotations.SerializedName;

public class DatabaseModel {
    @SerializedName("db_host")
    public String dbHost;

    @SerializedName("db_name")
    public String dbName;

    @SerializedName("db_username")
    public String dbUsername;

    @SerializedName("db_password")
    public String dbPassword;
}

File: src/main/java/com/basecolon/uisless/model/DatabaseModel.kt
package com.basecolon.uisless.model

import com.google.gson.annotations.SerializedName

data class DatabaseModel(
    @SerializedName("db_host")
    var dbHost: String? = null,
    @SerializedName("db_name")
    var dbName: String? = null,

    @SerializedName("db_username")
    var dbUsername: String? = null,
    @SerializedName("db_password")
    var dbPassword: String? = null
)
